eng/Packages.Data.props

```diff
@@ -9,11 +9,12 @@
   <ItemGroup>
     <PackageReference Update="ApprovalTests" Version="3.0.22" />
     <PackageReference Update="ApprovalUtilities" Version="3.0.22" />
-    <PackageReference Update="Azure.AI.TextAnalytics" Version="1.0.0" />
+    <PackageReference Update="Azure.AI.FormRecognizer" Version="3.0.0" />
```

Comment:
These changes are being sync'd from `master`, based on commit `0abf2a2ee7104bc0922775373f34aa197554a95b`, reflecting the `Azure.Core` release.   

I've confirmed none of the changes will impact the Event Hubs release in an unexpected or negative way.

Reply:
These were causing conflicts elsewhere in the tree due to changes with the `BinaryData` type in `Azure.Core.Experimental`;  I've reverted this and used local version overrides for Event Hubs to avoid orthogonal updates elsewhere.